pkg/proto/configuration/http/http.proto

```diff
@@ -177,3 +185,12 @@ message AcceptHeaderAuthenticationPolicy {
   // provided.
   AuthenticationPolicy policy = 2;
 }
+
+message RemoteHttpRequestAuthenticationPolicy {
+  // The remote authenticator to grant or deny access the HTTP request.
+  buildbarn.configuration.grpc.RemoteAuthenticationPolicy backend = 1;
```

Comment:
Shouldn't this message live in buildbarn.configuration.auth, because it's independent of gRPC/HTTP?

Reply:
This creates a circular dependency between `grpc.proto` and `auth.proto`: `grpc.ServerConfiguration -> grpc.AuthenticationPolicy -> auth.RemoteAuthenticator -> grpc.ClientConfiguration`. If we split into `grpc_client.proto` and `grpc_server.proto`, it will work. Shall I do that?